admin-client/src/Router.svelte

```diff
@@ -66,7 +66,9 @@
   page('/organizations/new', queryString, render(Pages.Organization.New));
   page('/organizations/:id', queryString, render(Pages.Organization.Show));
   page('/organizations/:id/edit', queryString, render(Pages.Organization.Edit));
-  page('/organizations-report', queryString, render(Pages.Organization.Report));
+  page('/reports/organizations', queryString, render(Pages.Organization.OrgsReport));
```

Comment:
This is just a question -- when we remove routes that have previously been public (possibly used/bookmarked by customers), have we ever (or could we) define redirects to the next closest route?

Reply:
Not sure as a general question. But in _this_ case this is admin-only, only usable by Pages operators, only called by the admin interface, and only in existence since I [added it](https://github.com/cloud-gov/pages-core/pull/4145) in May.